### PR TITLE
change values for IDAM API URL

### PR DESCRIPTION
**Pull request by ngul**

### CODE REVIEW

File: charts/ccd-api-gateway-web/values.aat.template.yaml [hunk 1]
```diff
@@ -1,8 +1,11 @@
 nodejs:
   environment:
-    IDAM_OAUTH2_TOKEN_ENDPOINT: https://preprod-idamapi.reform.hmcts.net:3511/oauth2/token
-    IDAM_OAUTH2_LOGOUT_ENDPOINT: https://preprod-idamapi.reform.hmcts.net:3511/session/:token
-    IDAM_BASE_URL: https://preprod-idamapi.reform.hmcts.net:3511
+    #IDAM_OAUTH2_TOKEN_ENDPOINT: https://preprod-idamapi.reform.hmcts.net:3511/oauth2/token
+    IDAM_OAUTH2_TOKEN_ENDPOINT: https://idam-api.aat.platform.hmcts.net
```
Comment (dwaynebailey) [inline charts/ccd-api-gateway-web/values.aat.template.yaml]: Missing the trailing bits

File: charts/ccd-api-gateway-web/values.aat.template.yaml [hunk 2]
```diff
@@ -1,8 +1,11 @@
...
+    #IDAM_OAUTH2_LOGOUT_ENDPOINT: https://preprod-idamapi.reform.hmcts.net:3511/session/:token
+    IDAM_OAUTH2_LOGOUT_ENDPOINT: https://idam-api.aat.platform.hmcts.net
```
Comment (dwaynebailey) [inline charts/ccd-api-gateway-web/values.aat.template.yaml]: Missing trailing bits

File: charts/ccd-api-gateway-web/values.aat.template.yaml (hunk 1 above)
Comment (ngul) [reply charts/ccd-api-gateway-web/values.aat.template.yaml]: fixed

File: charts/ccd-api-gateway-web/values.aat.template.yaml (hunk 2 above)
Comment (ngul) [reply charts/ccd-api-gateway-web/values.aat.template.yaml]: fixed

**Pull request merged**